recipes/lodepng/all/conanfile.py

```diff
@@ -35,6 +35,14 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+        try:
+            del self.settings.compiler.libcxx
+        except Exception:
+            pass
+        try:
+            del self.settings.compiler.cppstd
+        except Exception:
+            pass
```

Comment:
No, lodepng is a C++ library. Could you open an issue? I can cross-build lodepng.